scripts/tests/chiptest/__init__.py

```diff
@@ -160,6 +160,55 @@ def _GetInDevelopmentTests() -> Set[str]:
     }
 
 
+def _GetDarwinFrameworkToolUnsupportedTests() -> Set[str]:
+    """Tests that fail in darwin-framework-tool for some reason"""
+    return {
+        "DL_LockUnlock",  # Events error.
+        "Test_AddNewFabricFromExistingFabric",  # darwin-framework-tool does not support the GetCommissionerRootCertificate command.
+        "TestAttributesById",
+        "TestBasicInformation",  # Test Failure : The test expects the "UNSUPPORTED_WRITE" error but the "FAILURE" error occured
+        "TestClusterComplexTypes",
+        "TestClusterMultiFabric",
+        "TestCommandsById",  # Test Failure : The test expects the "UNSUPPORTED_ENDPOINT" error but the "FAILURE" error occured.
+        "TestCommissioningWindow",
+        "TestDiscovery",  # darwin-framework-tool does not support dns-sd commands.
+        "TestEvents",
+        "TestEventsById",
+        "TestGroupMessaging",  # darwin-framework-tool does not support group commands.
+        "TestReadNoneSubscribeNone",  # darwin-framework-tool does not supports those commands.
+
+        "Test_TC_ACE_1_6",  # darwin-framework-tool does not support group commands.
+        "Test_TC_ACL_2_5",  # Events error.
+        "Test_TC_ACL_2_6",  # Events error.
+        "Test_TC_ACL_2_7",  # Events error.
+        "Test_TC_ACL_2_8",  # Events error.
+        "Test_TC_ACL_2_9",  # Events error.
+        "Test_TC_ACL_2_10",  # Events error.
+        "Test_TC_BINFO_2_1",  # Test Failure : The test expects the "UNSUPPORTED_WRITE" error but the "FAILURE" error occured
```

Comment:
Let's diagnose why this is. Is it possible our existing YAML in some cases "passes" because UnsupportedWrite is a failure, but the actual device reported FAILURE (i.e. we are smoothing over an SDK bug) ?

Reply:
It happens because we are trying to do a "force-write" which is not a command that is supported by the "darwin-framework-tool" yet. So the runner fails to send the command - I have a work in progress patch around fwiw, it does not work fully because of attributes of type struct but at some point it will work.

Reply:
Then the comment here should say:
```suggestion
        "Test_TC_BINFO_2_1",  # darwin-framework-tool does not support writing readonly attributes by name
```
or so.